lineage/data/Lineage_collections.py

```diff
@@ -8,38 +8,42 @@
 desired_num_states = 2
 E = [StateDistribution() for _ in range(desired_num_states)]
 
-
-def popout_single_lineages(lineages):
-    """ To remove lineages with cell numbers <= 5. """
-    trimed_lineages = []
-    for cells in lineages:
-        if len(cells) < 5:
-            pass
-        else:
-            trimed_lineages.append(cells)
-    assert len(trimed_lineages) > 0
+def popout_single_lineages(lineages):	
+    """ To remove lineages with cell numbers <= 5. """	
+    trimed_lineages = []	
+    for cells in lineages:	
+        if len(cells) < 5:	
+            pass	
+        else:	
+            trimed_lineages.append(cells)	
+    assert len(trimed_lineages) > 0	
     return trimed_lineages
 
 
 # -- Lapatinib control
 
 lap01 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00601_A5_1_V5.xlsx")]
+lap012 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00601_A5_2_V4.xlsx")]
 lap02 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00701_A5_1_V4.xlsx")]
 lap03 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00801_A5_1_V4.xlsx")]
 
-Lapatinib_Control = lap01 + lap02 + lap03
+Lapatinib_Control = lap01 + lap012 + lap02 + lap03
+# used for when we want to writes states instead of phase lengths
+len_lp_cntr = [len(lap01), len(lap012), len(lap02), len(lap03)]
 
 # -- LAPATINIB 25 uMolars
 
 lapb63 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00601_B6_1_V4.xlsx")]
-lapb64 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00701_B6_1_V4.xlsx")]
-lapb65 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00801_B6_1_V4.xlsx")]
-# replicates
 lapb632 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00601_B6_2_V4.xlsx")]
+lapb64 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00701_B6_1_V4.xlsx")]
```

Comment:
Why does this require changes to the file import?

Reply:
I made changes to the order of appending them so I won't lose track of the files that which is which. Also found one or two things needed modifications, I guess in one of them, I saw a repetition of the same file so I changed it, and in another case, found that I hadn't included a file or two, and added them.